bitnami/mongodb/templates/arbiter/headless-svc.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ include "mongodb.fullname" . }}-arbiter-headless
+  name: {{ include "mongodb.arbiterservicename" . }}
```

Comment:
Change arbiterservicename to arbiter.service.nameOverride

Reply:
Fixed